hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -727,6 +728,11 @@ public class HoodieWriteConfig extends HoodieConfig {
    */
   public static final String SPARK_SQL_MERGE_INTO_PREPPED_KEY = "_hoodie.spark.sql.merge.into.prepped";
 
+  /**
+   * An internal config referring to fileID encoding. 0 refers to UUID based encoding and 1 refers to raw string format(random string).
+   */
+  public static final String WRITES_FILEID_ENCODING =  "_hoodie.writes.fileid.encoding";
```

Comment:
how about `_hoodie.record.index.fileid.encoding`? This is going to be used on. the read path too. Also, can we make it clear in the javadoc that it is for record index.

Reply:
I just want to generalize this. we could think of extending FILES partition to store the fileId encoding as well. So, just to keep this a generic concept in hudi to deduce how fileIds are generated. 
thats why placed the config in HoodieWriteConfig. 
let me know wdyt

Reply:
sounds good.